js/adapt-contrib-slider.js

```diff
@@ -553,6 +553,30 @@ define([
             }
         },
 
+	getInteractionObject: function() {
```

Comment:
I think this would benefit from a comment along the lines of the two functions below this one